### ISSUE TITLE
Tab and TabButton should require an aria-controls prop

### ISSUE THREAD

**Issue by exoen** [SpareBank1/designsystem#626]
#### Which package is this issue related to?
ffe-tabs-react
#### Describe your issue (screenshots welcome!)
The tab and tabButton should require `aria-controls` so that consumers provide information of which element it controls to help the experience when using screen readers.
#### What is the expected behavior? 
aria-controls should be required
#### What is the actual behavior?
aria-controls is not required and must be set manually by the consumers

**Comment by ivarni** [comment]
I vote that we don't release this as a breaking change, since the only effect it has is to print a warning to the console when in dev-mode.

It can be a minor fix. I can do it unless you want to.

**Comment by exoen** [comment]
I don't mind changing the commit message =)

My initial thought to make this a breaking change is because developers that use the current package that has manually placed `aria-control` correctly suddenly gets a warning in the console for each tab used, which could be many warnings.

But this is really up to you.

Should remove the breaking change in the commit message?

**Comment by ivarni** [comment]
I've updated the review on the PR, my bad for being a bit quick there.

All we need to do is to indicate via `propTypes` that `aria-controls` is a required prop, there's no need for aliasing it and therefore it doesn't need to be breaking.

See comment on #627
